src/main/java/org/springframework/cloud/scheduler/spi/kubernetes/KubernetesSchedulerProperties.java

```diff
@@ -16,9 +16,14 @@
 
 package org.springframework.cloud.scheduler.spi.kubernetes;
 
+import io.fabric8.kubernetes.api.model.Volume;
```

Comment:
understood this was copied from deployer, but i would prefer that we do not expose fabric8 types via property classes, rather wrap them similar to say `StatefulSet`, `Toleration`, etc as in the deployer